src/python/pants/backend/jvm/tasks/jvm_compile/execution_graph.py

```diff
@@ -4,9 +4,10 @@
 
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-import Queue as queue
+import queue
```

Comment:
So, the interesting datapoint is:
https://github.com/timothycrosley/isort/blob/4.2.5/isort/settings.py#L53-L64

Modern isort has both `queue` and `Queue`:
https://github.com/timothycrosley/isort/blob/4.3.4/isort/settings.py#L58-L89

So this does look like an environment leak of some sort - which would be Pants fault. Digging a bit, but - for sanity sake - can you confirm `.isort.cfg` in your clone matches origin/master and is readable by the user you run `./pants` as?

Reply:
Another sanity check - you should see this:
```
$ ./pants options --scope=isort
isort.version = 4.2.5 (from HARDCODED)
```

Reply:
Thanks for looking into this John.

Results of sanity checks:
- `./isort.cfg' is the same. Not sure what you mean by it being accessible, but it has these permissions `-rw-r--r--`
- `./pants options --scope=isort` returns 4.2.5 as expected
- `settings.__file__` returns `'/Users/earellano/.pex/install/isort-4.2.5-py2.py3-none-any.whl.440f1368d2835debbb62deef09cb82e24bf351a2/isort-4.2.5-py2.py3-none-any.whl/isort/settings.pyc'`
- looks like my `PYTHONPATH contains` is similar to yours: 
```bash
pex: Found site-library: /usr/local/lib/python2.7/site-packages
pex: Found site-library: /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/site-python
pex: Found site-library: /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages
pex: Not a tainted path element: /Users/earellano/.cache/pants/bin/isort/4.2.5/isort.pex/.bootstrap
pex: Not a tainted path element: /Users/earellano/.cache/pants/bin/isort/4.2.5/isort.pex
pex: Not a tainted path element: /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python27.zip
pex: Not a tainted path element: /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7
pex: Not a tainted path element: /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-darwin
pex: Not a tainted path element: /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac
pex: Not a tainted path element: /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac/lib-scriptpackages
pex: Not a tainted path element: /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-tk
pex: Not a tainted path element: /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-old
pex: Not a tainted path element: /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload
pex: Tainted path element: /usr/local/lib/python2.7/site-packages
pex: Scrubbing from user site: /Users/earellano/Library/Python/2.7/lib/python/site-packages
pex: Scrubbing from site-packages: /usr/local/lib/python2.7/site-packages
pex: Scrubbing from path_importer_cache: /usr/local/lib/python2.7/site-packages
pex: Scrubbing from path_importer_cache: /Users/earellano/Library/Python/2.7/lib/python/site-packages
pex: Activating PEX virtual environment from /Users/earellano/.cache/pants/bin/isort/4.2.5/isort.pex :: Searching dependency cache: /Users/earellano/.cache/pants/bpex: Activating PEX virtual environment from /Users/earellano/.cache/pants/bin/isort/4.2.5/isort.pex :: Searching dependency cache: /Users/earellano/.cache/pants/bpex: Activating PEX virtual environment from /Users/earellano/.cache/pants/bin/isort/4.2.5/isort.pex :: Searching dependency cache: /Users/earellano/.cache/pants/bpex: Activating PEX virtual environment from /Users/earellano/.cache/pants/bin/isort/4.2.5/isort.pex :: Resolving isort==4.2.5                                     pex: Activating PEX virtual environment from /Users/earellano/.cache/pants/bin/isort/4.2.5/isort.pex: 15.7ms
pex:   Searching dependency cache: /Users/earellano/.cache/pants/bin/isort/4.2.5/isort.pex/.deps: 3.0ms
pex:     Adding setuptools 5.4.1: 0.2ms
pex:     Adding isort 4.2.5: 0.2ms
pex:   Resolving isort==4.2.5: 7.4ms
pex:   Resolving setuptools==5.4.1: 4.3ms
pex:   Activating isort 4.2.5: 0.4ms
pex:     Adding sitedir: 0.2ms
pex:   Activating setuptools 5.4.1: 0.3ms
pex:     Adding sitedir: 0.2ms
pex: PYTHONPATH contains:
pex:   * /Users/earellano/.cache/pants/bin/isort/4.2.5/isort.pex/.bootstrap
pex:     /Users/earellano/.cache/pants/bin/isort/4.2.5/isort.pex
pex:   * /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python27.zip
pex:     /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7
pex:     /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-darwin
pex:     /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac
pex:     /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac/lib-scriptpackages
pex:     /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-tk
pex:   * /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-old
pex:     /usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload
pex:     /Users/earellano/.pex/install/isort-4.2.5-py2.py3-none-any.whl.440f1368d2835debbb62deef09cb82e24bf351a2/isort-4.2.5-py2.py3-none-any.whl
pex:     /Users/earellano/.pex/install/setuptools-5.4.1-py2.py3-none-any.whl.77dd06420042734134bc9f929de0433f5c64ec67/setuptools-5.4.1-py2.py3-none-any.whl
pex:   * - paths that do not exist or will be imported via zipimport
pex: PEX_INTERPRETER specified, dropping into interpreter
```